src/main/java/core/basesyntax/dao/StoreDao.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.dao;
+
+import java.util.Map;
+
+public interface StoreDao {
+    void add(String key, Integer value);
+
+    Map<String, Integer> getStorage();
+
+    void clean();

```

Comment:
Its will be better if all your methods will return some type (for instanse boolean). Its helps you while testing your code.